tests/conftest.py

```diff
@@ -0,0 +1,34 @@
+from sqlalchemy import create_engine
```

Comment:
it'd be nice to add a comment to mention this file is a pytest configure file :-)

Reply:
Yes I agree, if you are not familiar with pytest this is confusing. Added a doctsring on top of the file.